### PR TITLE
Add usbsdmux support

### PR DESCRIPTION
**Pull request by jan-kiszka**
See patch 1 for details.

@fmoessbauer

### CODE REVIEW

**Comment by chombourger** [comment]
thank you! I was hoping to get one of these for myself. lucky you :)

**Comment by chombourger** [comment]
LGTM. Triggered our CI checks before merging.

**Comment by jan-kiszka** [comment]
Wait, I'm missing docs, and I just realized that I still hard-coded `/dev/sg0` - one minute...

**Comment by jan-kiszka** [comment]
Now it should be fine.

**Comment by chombourger** [comment]
> Now it should be fine.

thanks for taking the time to augment the documentation. These are good additions

**Comment by chombourger** [comment]
Looks like the spellcheck isn't happy about `distros`. Replace with `distributions` to keep the guy happy?

File: docs/config.rst [hunk 1]
```diff
@@ -388,6 +388,21 @@ a SD card between the DUT and host. The following settings are supported:
   Identifier of the sdmux/sdwire device to use. Use ``sd-mux-ctrl`` to list
   available devices. When not specified, the first device is auto-detected.
 
+``usbsdmux`` driver settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``usbsdmux`` driver supports SD card switcher based on the Microchip
+USB2642 card reader chip. A tool with this name is available for several
+distros and via pip. The following settings are supported:
```
Comment (chombourger) [inline docs/config.rst:396]: distros causes a spellcheck error in the CI. Change to distributions?

**Review by chombourger — APPROVED** [review]

**Pull request merged**